docs/chart_template_guide/variables.md

```diff
@@ -114,6 +114,7 @@ metadata:
     # I cannot reference .Chart.Name, but I can do $.Chart.Name
     helm.sh/chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     app.kubernetes.io/instance: "{{ $.Release.Name }}"
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
```

Comment:
Can you please add a comment above this to advice people to use the `app.kubernetes.io/version` if there is an `AppVersion` in their `Chart.yaml` file.